amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/Scheduler.java

```diff
@@ -345,6 +358,30 @@ void initialize() {
         }
     }
 
+    @VisibleForTesting
+    boolean waitAndCheckIfLeaseTableIsReady() throws InterruptedException,
```

Comment:
This can be named better. Like shouldInitiateLeaseSync() ?

Reply:
Addressed